web/src/components/tasks/government-contracting/GovernmentContractingPaginator.test.tsx

```diff
@@ -0,0 +1,46 @@
+import { GovernmentContractorPaginator } from "@/components/tasks/government-contracting/GovernmentContractingPaginator";
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
+
+describe("<GovernmentContractorPaginator />", () => {
+  it("renders first page without clicking stepper", async () => {
+    render(<GovernmentContractorPaginator />);
+    await waitFor(() => {
+      expect(screen.getByTestId("njstart-registration")).toBeInTheDocument();
+    });
+  });
+
+  it("renders NJ Start Page", async () => {
```

Comment:
note: update test so we navigate away from the first page and then navigate back as this currently is the same as the test above expect with a click

Reply:
done